deeplake/auto/unstructured/coco/convert.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+from typing import Dict, Any, Optional
+
+from deeplake.core.tensor import Tensor
+
+
+def coco_to_deeplake(
+    coco_key: str,
+    value: Any,
+    destination_tensor: Tensor,
+    category_lookup: Optional[Dict] = None,
+):
+    """Takes a key-value pair from coco data and converts it to data in Deep Lake compatible format"""
+    dtype = destination_tensor.meta.dtype
+
+    if coco_key == "bbox":
+        assert len(value) == 4
+        return np.array(value, dtype=dtype)
+    elif coco_key == "segmentation":
+        try:
+            # Currently having only ONE polygon is supported.
+            # Multiple polygons are under same label, but there is only a single bbox.
+            # Can not think of a way to support multiple polygons for same label on the same image, other than converting to a mask.
+            return np.array(value[0], dtype=dtype).reshape(
+                (len(value[0]) // 2), 2
+            )  # Convert to array of x-y coordinates
+        except KeyError:
+            """KeyError happens if the value is NOT a list of polygons.
+            Returning None does not work, as polygon is erroring out during shape validation.
```

Comment:
are you saying that sending None to polygon tensors doesn't work? This should be fixed. Can you create a min reproducible script and add it to backlog?

Reply:
In the context of ingest_coco, the issue is gone in case of a wrong value exception is thrown. In general, it is still there, but I can not reproduce it. Although I have a stack-trace from errors I encountered.